qutebrowser/mainwindow/tabbedbrowser.py

```diff
@@ -527,7 +527,9 @@ def _get_new_tab_idx(self, related):
     def _update_favicons(self):
         """Update favicons when config was changed."""
         for i, tab in enumerate(self.widgets()):
-            if config.val.tabs.favicons.show:
+            if (config.val.tabs.favicons.show == 'always' or
```

Comment:
These couple of lines are repeated several times, maybe we could make a `show_favicon` helper method either here (taking a tab) or a method in tab itself, to make this a bit cleaner?